VK_V1.0/VK_V1.0/Servics/PromiseNetworkService.swift

```diff
@@ -0,0 +1,49 @@
+// PromiseNetworkService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+import PromiseKit
+
+/// Сервис для получения данных друзей
```

Comment:
```suggestion
/// Сервис для получения данных друзей
```
Если только для друзей тогда название класса должно соответствовать